src/m_cheat.c

```diff
@@ -665,7 +665,6 @@ static void cheat_clev(char *buf)
 // killough 2/7/98: simplified using dprintf and made output more user-friendly
 static void cheat_mypos()
 {
-  plyr->cheats &= ~CF_RENDERSTATS;
```

Comment:
Should IDMYPOS just toggle the regular map coords widget?

Reply:
`IDMYPOS` is bad, isn't this here for historical reasons?

Reply:
IDMYPOS has its precision ramped up to the max. AFAIR this was due to a request from Looper when practicing certain glides with Crispy Doom. Not sure how useful it still is today.